server/src/entities/Error/AuthorizationError.ts

```diff
@@ -0,0 +1,8 @@
+export class AuthorizationError extends Error {
+    public statusCode: number;
+    public message: string;
+    constructor(statusCode: number, message: string) {
+        super(message);
+        this.statusCode = statusCode;
+    }
+}
```

Comment:
Nit pick but id probably just simplify this
```suggestion
export class AuthorizationError extends Error {
    public readonly statusCode = 401;
    constructor() {
        super("User not authorized");
    }
}
```